client_test.go

```diff
@@ -211,7 +211,7 @@ func TestClientBroadcast(t *testing.T) {
 		startEchoActor(fmt.Sprintf("echo-%d", i))
 	}
 
-	msg := &EchoMsg{"lol"}
```

Comment:
The new protobuf generated code includes more than one field on EchoMsg, so we have to qualify these now. Really we should always qualify them. Why does go even support not doing so.

Reply:
Yeah this is a good change either way!

Reply:
@vitaminmoo I think they allow them for in-line struct definitions (not sure if that's the right term):

```go
testCases := []struct{
	Name string
	WantErr bool
}{
	{"hello", false},
}
```

Agreed though, it's a good change!

Reply:
@mattayes Yeah, but those are a detectably different structure - To the point that `go fmt` will automatically remove them if you want (`-s` flag).